### ISSUE TITLE
NoClassDefFoundError: org/apache/commons/configuration/Configuration

### ISSUE THREAD

**Issue by elisabetao** [AbsaOSS/spline-spark-agent#219]
Hi,

When building the latest Spline Spark agent with Spark 3, and running a test I get
```
NoClassDefFoundError: org/apache/commons/configuration/Configuration
```
similar to this issue: https://www.gitmemory.com/issue/AbsaOSS/spline/870/823610404.
Due to this one needs to specify  
```
"spark.jars.packages": "commons-configuration:commons-configuration:1.10, za.co.absa.spline.agent.spark:spark-3.0-spline-agent-bundle_2.12:0.7.0-SNAPSHOT"
```
instead of 
```
"spark.jars.packages": "za.co.absa.spline.agent.spark:spark-3.0-spline-agent-bundle_2.12:0.7.0-SNAPSHOT"
```

Do you have any suggestions? Thanks a lot

**Comment by elisabetao** [comment]
I am guessing the 
```xml
<dependency>
    <groupId>commons-configuration</groupId>
    <artifactId>commons-configuration</artifactId>
    <version>1.10</version>
</dependency>
```
 should be in core pom.xml.

**Comment by wajda** [comment]
What exact version of Spark are you using?
I've checked 3.0.2 and 3.1.1 distributions and both contain `commons-configuration` library in them.

**Comment by wajda** [comment]
fixed in #248
